stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
Yes, I think so. It doesn't hurt if you still want to use cabal, and there is an open issue against the parent repo to support stack...